mockgen/mockgen.go

```diff
@@ -316,7 +317,11 @@ func (g *generator) Generate(pkg *model.Package, outputPkgName string, outputPac
 	g.p("// Generated by this command:")
 	g.p("//")
 	// only log the name of the executable, not the full path
-	g.p("//\t%v", strings.Join(append([]string{filepath.Base(os.Args[0])}, os.Args[1:]...), " "))
+	name := filepath.Base(os.Args[0])
+	if runtime.GOOS == `windows` && strings.HasSuffix(name, `.exe`) {
+		name = strings.TrimSuffix(name, `.exe`)
```

Comment:
Why not use normal quotation marks?

Reply:
It's a habit from the codebase I work on day-to-day. I'll change it.